data/darktableconfig.xml.in

```diff
@@ -1011,10 +1011,16 @@
     <name>plugins/darkroom/hide_default_presets</name>
     <type>bool</type>
     <default>false</default>
-    <shortdescription>hide built-in presets</shortdescription>
+    <shortdescription>hide built-in presets for image operations</shortdescription>
     <longdescription>hides built-in presets of modules in both presets and favourites menu.</longdescription>
   </dtconfig>
-
+  <dtconfig prefs="misc" section="interface">
```

Comment:
this one should be in lighttable section.

Reply:
done